src/components/profile/components/Leaderboard.tsx

```diff
@@ -1,10 +1,13 @@
 import React, { useState } from "react";
+import LeaderboardRow from "./LeaderboardRow";
 
 const Leaderboard = () => {
     const [selectedIndex, setSelectedIndex] = useState(0);
+
     const handleClick = (index) => {
-        setSelectedIndex(index)
-      }
+        setSelectedIndex(index);
+    };
+
     const LeaderBoardData = [
         { id: '288', name: 'Ruchi Tripathi', reward: '213' },

```

Comment:
Make an Interface for this data, don't just assign data directly, in future when we integrate API we can update the interface too.

And keep practice for having these constants and mocks in common separate file. If you want to keep it here, then keep it outside `Leaderboard` function, this are reassigning on every re-render.